### ISSUE TITLE
DONATION PROBLEMS!!!!

### ISSUE THREAD

**Issue by Ohpapatony** [dalaranwow/dalaran-wow#4811]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I put my credit card information in I have sufficient funds in my account and the payment isn't going through.
**Expected behavior**: Tell us what should happen instead
I should be able to donate to this website and receive my 22 donation points as promised
**Steps to reproduce the problem**:

1. give me my donation points
2.  process the payment accurately 
3. I just wanna play

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by warcraftgeekygamer** [comment]
Are you 100% absolutely sure that you have sufficient funds on your credit card? You can always call the bank who issued that credit card to verify whether or not you have available funds. Just sayin.

**Comment by Rushor** [comment]
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
